packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/index.md

```diff
@@ -44,55 +44,39 @@ The SDK represents the sign-in flow using a number of different objects:
 </div>
 
 - **Response:** The top-level object that represents a step and contains all the other objects. It includes a property that indicates a successful sign-in and functions for cancelling the sign-in flow, or retrieving the access token after the sign-in flow succeeds. A response may contain multiple authenticators and remediations.
-- **Remediation:** Represents the main user actions for a step, such as enrolling in an authenticator or entering an OTP. It includes a function for requesting the next step in the flow.
+- **Remediation:** Represents the main user actions for a step, such as enrolling in an authenticator or entering an OTP. In the Swift SDK for iOS, it also includes the function for requesting the next step in the flow.
 - **Authenticator:** Represents an authenticator that's used to verify the identity of a user, such as Okta Verify.
-- **Method:** Represents a channel for an authenticator, such as using SMS or voice for an authenticator that uses a phone. An authenticator may have zero or more.
+- **Method:** Represents a channel for an authenticator, such as using SMS or voice for an authenticator that uses a phone. An authenticator may have zero or more methods.
 - **Capability:** A user action associated with a remediation, authenticator, or method, such as requesting a new OTP or a password reset.
```

Comment:
A capability can also indicate additional data associated with an authenticator that isn't always present. For example, a Password authenticator may sometimes return "PasswordSettings" to describe the password complexity requirements (usually only available when setting a new password). Another useful one is the "Profile" capability, which returns additional display information for the authenticator that can be used to render a UI (such as the redacted email address or phone number used within an authenticator).

Reply:
How's this?

> - **Capability:** A user action or additional information that is associated with a remediation, authenticator, or method, such as requesting a new OTP or a redacted email.